src/services/fetchmovies.js

```diff
@@ -0,0 +1,11 @@
+import { checkError, client } from './client';
+
+export async function fetchMovies(query) {
+  const response = await client.from('movies').select('*');
+  if (query) {
+    const response = await client.from('movies').select('*').ilike('title', `%${query}%`);
+    return checkError(response);
+  }
+  //   console.log('Movies response', checkError(response));
+  return checkError(response);
+}
```

Comment:
Awesome work getting the query to render! Excellent job checking for a query with an if statement and rendering the results.